README.md

```diff
@@ -116,7 +116,7 @@ This will bring up etcd listening on port 2379 for client communication and on p
 Next, let's set a single key, and then retrieve it:
 
 ```
-etcdctl put mykey "this is awesome"
+etcdctl set mykey "this is awesome"
```

Comment:
@JJ thanks but `put` is correct (it has replaced set) - https://github.com/etcd-io/etcd/tree/master/etcdctl#put-options-key-value I am closing this PR. Thanks!